pages/projects/[id]/tasks/[taskId]/index.js

```diff
@@ -4,13 +4,9 @@ import styled from "styled-components";
 
 import Error from "@/components/Error";
 import Heading from "@/components/Heading";
-import { BackLink } from "@/components/Buttons";
+import { BackLink, EditLink } from "@/components/Buttons";
 import { DeleteButton } from "@/components/Buttons";
-
-const StyledContainer = styled.div`
-  padding-inline: 1rem;
-  margin-top: 1.5rem;
-`;
+import { StyledButtonContainer, StyledToolBar } from "../..";
 
 const StyledSection = styled.section`
```

Comment:
Just something to think about, if you are already importing styles from another place, why not move this styling to it's own file as well? Would keep the file more clean and give it a coherent look :)